_posts/2018-06-27-inception-pipelines-pt4.markdown

```diff
@@ -0,0 +1,171 @@
+* [API Gateway Private Integrations](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-private-integration.html)
+
+The last two points make it easier and safer for a service from the Shared Service account to be consumed by VPC's in other accounts through VPC Peering.
+
+#### Sandbox AWS Account
+
+The Sandbox AWS Account is used by developers as a safe place to try out all the services that AWS provides. It is a volatile landscape with lots of things being created and destroyed all the time. Developers should have full organisational and AWS permissions here to experiment. To help prevent cost blowouts, AWS Budgets should be used to alert when the forecasted, and actual, spend hits predefined levels.
+
+It is also possible to define a Lambda in the root account that periodically deletes the Sandbox account and recreates it in a clean state.
+
+The account Inception Pipeline defines the roles that can be assumed, as well as setup AWS Budget alerts. When people are experimenting, they would have their own copy of the pipeline.
+
+### Managing Aspects
+
+Now we move into the interesting bits and answer the question "How do I structure my accounts\pipeline for my application?".
+
+#### All-In-One
+
+![cross account components]({{ site.url }}/img/inception-pipelines/part-4-all-in-one.png)
+
+In this model we just have a single AWS account with all the 'environments' living inside this one account. In here there is the usual account-level pipeline, and then at least one pipeline per aspect.
+
+Pros:
+
+* Everything is in one place making it easy to see where everything is and what the status is.
+* Deployments are potentially easier since you don't have to worry about crossing AWS account boundaries.
+
+Cons:
+
+* Everything is in one place so if your account is compromised then the blast radius is huge.
+* It would quickly become crowded and harder to manage resource name collisions.
+* Easy to make a mistake and accidentally delete production resources.
+* While it is possible to network segregate environments using VPC subnets, they are subjected to the Account Service Limit, and is also just plain harder to find stuff.
+
+#### Production / Non-Production
+
+![cross account components]({{ site.url }}/img/inception-pipelines/part-4-prod-non-prod.png)
+
+This model separates out the production environment into its own AWS account.
+
+Pros:
+
+* If non-production is compromised, then production is still safe.
+* Still has the quick feedback cycle on change to non-production environments as where you build the code is also the same AWS account that you run in.
+* Using new AWS services is easier as you don't need to adjust permission in multiple AWS accounts.
+
+Cons:
+
+* When you deploy into Production, it is a different process (i.e. across account boundaries), so the chance for something to go wrong increases.
```

Comment:
Here you capitalise Production, above (line 129) it's lower cased.

Additionally, in your i.e. bit do you mean _cross_ account boundaries